cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectS2ICluster.java

```diff
@@ -184,6 +201,11 @@ public ImageStream generateSourceImageStream() {
         sourceTag.setName(sourceImageTag);
         sourceTag.setFrom(image);
 
+        if (insecureSourceRepository)   {
+            sourceTag.setImportPolicy(new TagImportPolicyBuilder().withInsecure(true).build());
+            sourceTag.setReferencePolicy(new TagReferencePolicyBuilder().withType("Local").build());
```

Comment:
If you mean the OpenShift documentation, you can fined it [here](https://docs.openshift.org/3.9/dev_guide/managing_images.html#imagestream-tag-policies). The Local policy should helpo to avoid issues when the registry is not trusted by the Docker engine.

`insecure-source-repo` IMHO explains from user perspective quite well when to set this to true. SO I think it is a good name. I do not think that the option name should be named after what is does but rather after when it should be used. If we name it `source-image-stream-from-tag-insecure-import-policy-and-local-reference-policy` it would describe what it does but not when it should be used. 

But I'm open to suggestions ;-).